### DISCUSSION TITLE
What would be audio delay setting for 38 seconds?

### DISCUSSION THREAD

**Discussion by tack32** [smplayer-dev/smplayer#1030] [General]
I can't figure this out. The audio starts 38 seconds earlier than the video.  I tried 3800, 38000, -3800 and -38000 but that didn't get it right. Could someone say how much I should set the "audio delay" to make it match? Thanks!

**Comment by Raza-0007** [comment]
The default audio delay step for smplayer is 100 ms. So a 1000 ms is 1 sec. You need to type 38000 for delaying audio by 38 seconds. But I am not sure if smplayer is able to fix such a huge A/V desync. It is better to use some encoding utility to fix the video file itself.

~Raza

**Comment by tack32** [comment]
Actually, it's that the video of a concert didn't have any audio when I
downloaded the stream. The audio was on a separate stream. It wasn't
started exactly when the video started. So I just trying to watch it and
selecting "load external file" to try to get the sound near. Good to at
least know how to approach the number for the audio delay. Thanks.
